apis/v1alpha3/backendtlspolicy_types.go

```diff
@@ -102,19 +105,22 @@ type BackendTLSPolicyConfig struct {
 	//
 	// +kubebuilder:validation:MaxItems=8
 	// +optional
-	CACertRefs []v1beta1.LocalObjectReference `json:"caCertRefs,omitempty"`
+	CACertificateRefs []v1.LocalObjectReference `json:"caCertificateRefs,omitempty"`
 
-	// WellKnownCACerts specifies whether system CA certificates may be used in
+	// WellKnownCACertificates specifies whether system CA certificates may be used in
 	// the TLS handshake between the gateway and backend pod.
 	//
-	// If WellKnownCACerts is unspecified or empty (""), then CACertRefs must be
-	// specified with at least one entry for a valid configuration. Only one of
-	// CACertRefs or WellKnownCACerts may be specified, not both.
+	// If WellKnownCACertificates is unspecified or empty (""), then CACertificateRefs
+	// must be specified with at least one entry for a valid configuration. Only one of
+	// CACertificateRefs or WellKnownCACertificates may be specified, not both. If an
+	// implementation does not support the WellKnownCACertificates field or the value
+	// supplied is not supported, the Status Conditions on the Policy MUST be
+	// updated to include an Accepted: False Condition with Reason: Invalid.
 	//
-	// Support: Core for "System"
+	// Support: Implementation-specific
 	//
 	// +optional
-	WellKnownCACerts *WellKnownCACertType `json:"wellKnownCACerts,omitempty"`
+	WellKnownCACertificates *WellKnownCACertificatesType `json:"wellKnownCACertificates,omitempty"`
```

Comment:
Who defines this list of WellKnownCA certiticates? is it implementation specific?

Reply:
should this more explicit as foo.com/bar 

https://ccadb.my.salesforce-sites.com/mozilla/CACertificatesInFirefoxReport
https://developer.visa.com/pages/trusted_certifying_authorities

Reply:
yep, implementation specific, we should probably make this more clear. In many/most cases, this will depend on something like https://packages.debian.org/sid/ca-certificates being used by the Gateway, but there are lots of variations of what is trusted, or some may even choose to inject their own well known CA certs to a controller container to have them considered as "system", all of this is inherently implementation specific though.

Reply:
ok, is clarified in the documentation

>  There may be some variation in which system certificates are used by each implementation. Refer to documentation from your implementation of choice for more information.